scripts/example.ts

```diff
@@ -0,0 +1,144 @@
+import path from 'path';
+import { remove, pathExists } from 'fs-extra';
+import prompts from 'prompts';
+
+import { getOptionsOrPrompt } from './utils/options';
+import type { CLIStep } from './utils/cli-step';
+import { executeCLIStep } from './utils/cli-step';
+
+const frameworks = ['react'];
+const addons = ['a11y', 'storysource'];
+const examplesDir = path.resolve(__dirname, '../examples');
+
+async function getOptions() {
+  return getOptionsOrPrompt('yarn example', {
+    framework: {
+      description: 'Which framework would you like to use?',
+      values: frameworks,
+      required: true,
+    },
+    addon: {
+      description: 'Which extra addons (beyond the CLI defaults) would you like installed?',
+      values: addons,
+      multiple: true,
+    },
+    includeStories: {
+      description:
+        "Include Storybook's own stories (only applies if a react-based framework is used)?",
+    },
+    create: {
+      description: 'Create the example from scratch (rather than degitting it)?',
+    },
+    forceDelete: {
+      description: 'Always delete an existing example, even if it has the same configuration?',
```

Comment:
I wonder if this should be part of a validation like it's done [here](https://github.com/storybookjs/storybook/blob/tom/sb-568-script-asks-the-maintainer-which/code/lib/cli/src/repro.ts#L127) and only happen when the use case is actually true 🤔 

When reading "Always delete" it sounds to me that I will just chose this once and never again, so my answer will be saved somehow

Reply:
I think we need to hide these two options from the prompt. They are just CLI flags